ansible-ipi-install/roles/installer/templates/rhcos-image-md5sum.j2

```diff
@@ -0,0 +1 @@
+{{ rhcos_image_md5sum.stdout }}
```

Comment:
where does this get used?

Reply:
Good catch.  We no longer need this template.